### DISCUSSION TITLE
does skywalking oap will listen this port 9411

### DISCUSSION THREAD

**Discussion by marklinglon** [apache/skywalking#8020] [Q&A]
![image](https://user-images.githubusercontent.com/18097531/139013423-3966121a-e6cf-4fee-880f-a6aeccee1178.png)
does skywalking oap will listen this port 9411
SW_RECEIVER_ZIPKIN_HOST
SW_RECEIVER_ZIPKIN_PORT

**Comment by wu-sheng** [comment]
Set `SW_RECEIVER_ZIPKIN=default`.
